### PR TITLE
MPA-28: Create toolbar

### PR DESCRIPTION
**Pull request by DaniilGirskiy**

### CODE REVIEW

File: MinderaPeople/Screens/HomeView.swift [hunk 1]
```diff
@@ -89,6 +120,17 @@ struct HomeView: View {
...
+
+    private func icon(_ name: String) -> some View {
+        Image(name)
+            .resizable()
+            .scaledToFit()
+            .frame(width: 24, height: 24)
+    }
+    
+    private func text(_ text: String) -> some View {
+        Text(text)
```
Comment (pedroseruca) [inline MinderaPeople/Screens/HomeView.swift:132]: you should add font properties as specified by figma file

Comment (pedroseruca) [reply MinderaPeople/Screens/HomeView.swift:132]: already have that on designSystem as a method

File: MinderaPeople/Screens/HomeView.swift [hunk 2]
```diff
@@ -89,6 +120,17 @@ struct HomeView: View {
             LoginView(store: .init(initialState: .init(), reducer: LoginFeature()))
         }
     }
+
+    private func icon(_ name: String) -> some View {
+        Image(name)
+            .resizable()
+            .scaledToFit()
+            .frame(width: 24, height: 24)
```
Comment (pedroseruca) [inline MinderaPeople/Screens/HomeView.swift:128]: both image and text is missing the color for when it is selected and not selected

Comment (DaniilGirskiy) [reply MinderaPeople/Screens/HomeView.swift:128]: `Button(action: {
                    controller.selected = tab.element.selection
                }) {
                    VStack {
                        tab.element.icon
                        tab.element.text
                            .font(.system(size: 10))
                    }
                }
                .foregroundColor(controller.selected == tab.element.selection ? Color.indigo(._600) : Color.greyBlue(._500))`
                I've already have this information in DesignSystem repo. Should I pass it from View?

File: MinderaPeople/Screens/HomeView.swift (hunk 1 above)
Comment (DaniilGirskiy) [reply MinderaPeople/Screens/HomeView.swift:132]: about the fonts, I didn't find font 10 in Fonts.swift file
image

Should I add this one?

Comment (pedroseruca) [reply MinderaPeople/Screens/HomeView.swift:132]: I wouldn't... 10 px for a font is very very small... should be avoided IMO. Or we go for the 12 being so, or ask designer for clarification...

**Review by pedroseruca — APPROVED** [review]
Lets just wait for the release of MinderaDesignSystem to merge this one with the right package version

**Comment by pedroseruca** [comment]
Please update the Design system version to `0.0.2`

**Pull request closed without merging**